ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -102,6 +108,66 @@ def cleanup(cluster_name, cluster_id, upi=False):
         destroy_cluster(installer=oc_bin, cluster_path=cleanup_path)
 
 
+def cycle_nodes(cluster_path, action):
+    """
+     Start/Stop AWS nodes to save costs when not in use.
+
+     Args:
+        cluster_path(str): location of cluster path that has auth files
+        action (str): action to perform either start or stop
+
+    """
+    node_obj_file = os.path.join(cluster_path, NODE_OBJ_FILE)
+    nodes_file = os.path.join(cluster_path, NODE_FILE)
+    instance_file = os.path.join(cluster_path, INSTANCE_FILE)
+    if action == 'stop':
+        node_objs = get_node_objs()
+        kls = platform_nodes.PlatformNodesFactory()
```

Comment:
good idea to use this class as it will allow this functionality on future supported cloud providers, like Azure

Reply:
+1